.github/workflows/api_test.yml

```
@@ -0,0 +1,31 @@
+        options: >-
+          --health-cmd "curl -o /dev/null -f -s -w 'Status: %{http_code},\tTime: %{time_total}s' http://localhost:7860/"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-start-period 15s
+          --health-retries 15
+    steps:
+      - run: |
+          echo "Vocalizr available on localhost:${{ job.services.vocalizr.ports['7860'] }}"
+          curl -o /dev/null -f -s -w 'Status: %{http_code},\tTime: %{time_total}s' http://localhost:${{ job.services.vocalizr.ports['7860'] }}/
+      - name: Checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)